ui/packages/catalog-ui-search/src/main/webapp/js/extensions/marionette.ItemView.attachElContent.js

```diff
@@ -18,12 +18,14 @@ import React from 'react'
 const Parser = require('html-react-parser')
 import ExtensionPoints from '../../extension-points'
 
+const Providers = ExtensionPoints.providers
```

Comment:
We should leave this it existed so it has no chance of being stale.  If we pull it off into a variable and the ExtensionPoints update after (shouldn't technically happen, but could), then the providers will be out of date.